src/main/resources/spark_utils.py

```diff
@@ -0,0 +1,19 @@
+from pyspark import SparkContext
+from pyspark.sql import SparkSession
+
+spark = SparkSession.builder.getOrCreate()
```

Comment:
To locate the current active session, rather use:

```python
spark = pyspark.sql.SparkSession._instantiatedSession
```

In the event that the session does not exist (although unlikely), we would not want to create a new one using defaults since it will not work properly in our environment.